crates/ruff_linter/src/checkers/logical_lines.rs

```diff
@@ -15,13 +15,14 @@ use crate::rules::pycodestyle::rules::logical_lines::{
 use crate::settings::LinterSettings;
 
 /// Return the amount of indentation, expanding tabs to the next multiple of 8.
```

Comment:
Nit: mind updating this comment?

Reply:
Done in [ff9743d](https://github.com/astral-sh/ruff/pull/9506/commits/ff9743dea971b18c8cc8ded75e9937154ff78d40)